blocks/article-body-block/chains/article-body/default.jsx

```diff
@@ -32,6 +32,36 @@ const StyledLink = styled.a`
   color: ${(props) => props.primaryColor};
 `;
 
+// IMPROVEMENT: margin and margin bottom rem is reusable
+const FloatableImageContainer = styled.figure`
+  margin: 0 0 0 1rem;
+
+  @media screen and (min-width: 48rem) {
+    margin-bottom: 1.5rem;
+  }
+
+  ${({ allowedFloatValue }) => allowedFloatValue && css`
```

Comment:
The ticket mentions this should not be applied to "mobile" breakpoint. This is floating the image at all breakpoints

Reply:
updated

Reply:
my bad, think I didn't push the latest commits 🤦